lib/std/os/linux.zig

```diff
@@ -92,6 +91,8 @@ pub const pie = @import("linux/start_pie.zig");
 pub const BPF = @import("linux/bpf.zig");
 pub const IOCTL = @import("linux/ioctl.zig");
 pub const SECCOMP = @import("linux/seccomp.zig");
+pub const syscalls = @import("linux/syscalls.zig");
+pub const SYS = syscalls.SYS;
```

Comment:
I request to move the CPU architecture logic to here:

```suggestion
pub const SYS = switch (builtin.cpu.arch) {
```